### ISSUE TITLE
HiDPI support on wayland?

### ISSUE THREAD

**Issue by glyh** [vnotex/vnote#1991]
**Describe the bug**
VNote is blur under HiDPI wayland config. VNote在Wayland HiDPI配置下显示模糊。

**To Reproduce**
1. Use some wayland compositer (e.g.) sway with a HiDPI screen. 使用Wayland的Compositer(如Sway)， 以及HiDPI屏幕。
2. Open VNote. 打开VNote。

**Expected behavior**
There's no blur appearing. 没有模糊出现。

**Screenshots**

![image](https://user-images.githubusercontent.com/12870742/147515676-8f16a04f-8963-4521-ad43-8852658cf1fc.png)

**Desktop (please complete the following information):**
 - OS: `Linux arch 5.15.11-arch2-1 #1 SMP PREEMPT Wed, 22 Dec 2021 09:23:54 +0000 x86_64 GNU/Linux`
 - Compositer: `Sway, Wayland`

**Comment by tamlok** [comment]
Sway is a tile wm?

**Comment by glyh** [comment]
@tamlok Yes, you can refer to [Sway's Github Repo](https://github.com/swaywm/sway)
